### PR TITLE
[11.x] Symfony v7.0 support

### PR DESCRIPTION
**Pull request by driesvints**
Help test Symfony v7.0: https://github.com/symfony/symfony/issues/51563

Tasks:

* [x] Symfony 7 support for `laravel/prompts`: laravel/prompts#93
* [x] Symfony 7 support for `nunomaduro/termwind`: nunomaduro/termwind#176
* [x] Symfony 7 support for `fruitcake/php-cors`: fruitcake/php-cors#19
* [x] Fixes tests: symfony/symfony#51904

### CODE REVIEW

File: src/Illuminate/Http/Request.php [hunk 1]
```diff
@@ -28,6 +28,13 @@ class Request extends SymfonyRequest implements Arrayable, ArrayAccess
         Concerns\InteractsWithInput,
         Macroable;
 
+    /**
+     * The Laravel session instance.
+     *
+     * @var
```
Comment (Jubeki) [inline src/Illuminate/Http/Request.php:34]:
```suggestion
     * @var \Illuminate\Contracts\Session\Session
```

Comment (driesvints) [reply src/Illuminate/Http/Request.php]: Thanks!

File: src/Illuminate/Console/Command.php [hunk 2]
```diff
@@ -192,7 +192,7 @@ public function run(InputInterface $input, OutputInterface $output): int
      * @param  \Symfony\Component\Console\Output\OutputInterface  $output
      * @return int
```
Comment (datlechin) [inline src/Illuminate/Console/Command.php:193]:
```suggestion
```

Comment (crynobone) [reply src/Illuminate/Console/Command.php]: Thanks

**Comment by driesvints** [comment]
✅  Approved

**Pull request merged**